tools/compare/UnixRunner.cs

```diff
@@ -11,7 +11,11 @@ namespace compare
 	public class UnixRunner
 	{
 		public ProcessStartInfo info { get; }
+
 		ProcessStartInfo clientInfo;
+
+		public ProcessStartInfo infoAOT { get ; }
```

Comment:
Properties start with upper case letters.  Also, technically, it should be `InfoAot`, but I've always disliked that.